providers/yttcb/clusterbootstrap.yaml

```diff
@@ -1,7 +1,6 @@
 #@ load("@ytt:assert", "assert")
 #@ load("@ytt:data", "data")
 #@ load("@ytt:ip", "ip")
-#@ load("lib/helpers.star", "get_tkr_version_from_tkr_name")
```

Comment:
Wondering how this is currently invoked such that `lib/helpers.star` is loaded with `get_tkr_version_from_tkr_name` and not currently erroring. 🤔 

This path must be meaningful and this function must be loaded right now, even though this file is not using it.